tests/e2e/tools/lib/editor.js

```diff
@@ -54,20 +57,46 @@ async function addPostType({editor}, postType) {
 /**
  * Remove all Post Types from a Post
  *
- * @param {{Editor}} editor
+ * @param {{Page, Editor}} options - Page and Editor object
  */
-async function removeAllPostTypes({editor}) {
-  const editorSettings = await openComponentPanel({editor}, 'Post Types');
+async function removeAllPostTypes({page, editor}) {
+  const editorSettings = await openComponentPanel({page, editor}, 'Post Types');
   const buttons = await editorSettings.getByRole('button', {name: 'Remove Post Type'}).all();
   for (const button of buttons) {
     await button.click();
   }
 }
 
+/**
+ * Insert new block into page using the block inserter
+ *
+ * @param {Object} page             - The page object used to interact with the page.
+ * @param {Object} params           - Additional block's parameters.
+ * @param {Object} params.blockName - The name of the block.
+ * @param {Object} params.namespace - The namespace to search if it is needed.
+ * @return {Promise<void>}          - Playwright Locator
+ */
+const searchAndInsertBlock = async (page, {blockName, namespace = ''}) => {
+  // Can be also used the aria-label="Toggle block inserter"
```

Comment:
I've actually used the `Toggle block inserter` one in [this PR ](https://github.com/greenpeace/planet4-master-theme/pull/2420/files) because with the other approach I wasn't able to add several blocks in a row, so I would prefer this one. Good to create a reusable function though 👍